docs/compose.md

```diff
@@ -54,6 +54,7 @@ will provide a container `tritonserver` locally. You can access the container wi
 $ docker run -it tritonserver:latest
 ```
 
+Note: If `compose.py` is run on release versions older than r21.08, the resulting container will have DCGM version 2.2.8 installed. This may result in different GPU statistic reporting behavior.
```

Comment:
Same here. This should be added during codefreeze. Additionally won't 21.07 have 2.2.3 with your latest changes?